mesheryctl/internal/cli/root/system/channel_test.go

```diff
@@ -21,7 +21,7 @@ func SetupContextEnv(t *testing.T) {
 		t.Error("unable to locate meshery directory")
 	}
 	viper.Reset()
-	viper.SetConfigFile(path + "/../../../../pkg/utils/TestConfig.yaml")
+	viper.SetConfigFile(path + "/../../../../pkg/utils/fixtures/TestConfig.yaml")
```

Comment:
Don't shift the location of TestConfig.yaml if you need one for your fixture add one new config in the fixture directory